components/popover/stories/template.js

```diff
@@ -159,3 +171,39 @@ export const Template = ({
 		</div>
 	`;
 };
+
+export const SourcelessTemplate = ({
```

Comment:
Can you add some more context in a comment here about why this additional template is needed?

Reply:
Great callout on this! Turns out all I had to do was pass a null trigger where I didn't want a source button to show up (for Chromatic), so I've removed this code 👍